src/components/Redoc/Redoc.tsx

```diff
@@ -22,11 +23,15 @@ import {
 import { SearchBox } from '../SearchBox/SearchBox';
 import { StoreProvider } from '../StoreBuilder';
 import { VersionSelector } from '../VersionSelector';
+import { createGlobalStyle } from 'styled-components';
+import { globalStyles } from '../../globalStyles';
 
 export interface RedocProps {
   store: AppStore;
 }
 
+const GlobalCss = createGlobalStyle`${globalStyles}`;
```

Comment:
(Non-blocking observation) I noticed in the outputted HTML file that the new global styles can appear defined more than once, especially when dark mode is toggled (see video below; `:root` is initially defined twice, and then `.dark-mode`). 

I don't see this issue in dev mode though. Not sure if you're aware of anything that might be causing this, but figured I'd bring it up in case this is something that could become an issue.

https://github.com/user-attachments/assets/3274c00c-e68e-4895-9094-8d909ed7703f

Reply:
Good eye! Just found that it looks like i[t's a bug](https://github.com/styled-components/styled-components/issues/3076) with styled-components?? 
I agree that it's non-blocking. There are no adverse effects here on the UI or UX.

Reply:
Gotcha, good to know. Thanks for looking into this! I should've done my due diligence to look into this before commenting 🤦